src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/CommonRoutes.scala

```diff
@@ -76,7 +75,7 @@ private[routes] abstract class CommonRoutes(
   def routes: Route
 
   def create(schema: Ref): Route =
-    (post & noParameter('rev.as[Long]) & projectNotDeprecated & pathEndOrSingleSlash & hasPermission(writePermission)) {
+    (post & noParameter('rev.as[Long]) & projectNotDeprecated & pathEndOrSingleSlash & hasPermissions(writePermission)) {
```

Comment:
`hasPermissions(writePermission)` seems like redundant. We can just rename
writePermission -> write
readPermission -> read